test/load.test.ts

```diff
@@ -110,6 +110,40 @@ describe("load", function () {
         return expect(load("invalid-endpoint-url", credential)).eventually.rejectedWith("Invalid endpoint URL.");
     });
 
+    it("should filter by key and label, has(key) and get(key) should work", async () => {
+        const connectionString = createMockedConnectionString();
+        const settings = await load(connectionString, {
+            selectors: [{
+                keyFilter: "app.settings.*",
+                labelFilter: "\0"
+            }]
+        });
+        expect(settings).not.undefined;
+        expect(settings.has("app.settings.fontColor")).true;
+        expect(settings.has("app.settings.fontSize")).true;
+        expect(settings.has("app.settings.fontFamily")).false;

```

Comment:
Let's add testcases for `size`, `entries`, `keys`, `values` and iterator as well.

Reply:
added in 007c521